server/utils/interventionsFormError.ts

```diff
@@ -0,0 +1,64 @@
+import { InterventionsServiceError, InterventionsServiceValidationError } from '../services/interventionsService'
+import interventionsServiceErrorMessages from './interventionsServiceErrorMessages'
+import { FormValidationError } from './formValidationError'
+
+type InterventionsFormError = { type: 'general'; message: string } | ({ type: 'validation' } & FormValidationError)
+
+// Add new date fields here.
```

Comment:
Not sure what this comment is for?

Reply:
It was meant to be a piece of instruction for people wanting to know where to add further date fields in the future, but it does look a bit like a TODO. I'll get rid of it.